cpp/src/backends/core/backend.hh

```diff
@@ -67,8 +70,23 @@ class Backend {
   Backend() = default;
   virtual ~Backend() = default;
 
-  virtual bool Initialize(const std::string& model_dir) {
+  virtual bool Initialize(const std::string& model_dir,
+                          const std::string& metrics_config_path) {
```

Comment:
I don't understand why metrics_config_path should be passed at here.

Basically, metrics is loaded into caching at init time of model worker, and then the metrics caching is able to be accessed globally,